content/en/storage-providers/get-started/economics.md

```diff
@@ -17,24 +17,23 @@ There are two main types of rewards for their efforts: storage fees and block re
 
 ### Storage fees
 
-_PoSt (Proof-of-Spacetime)_ window checks are performed on 24 hour intervals across the network to ensure that storage providers are continuing to host their required sectors as normal. Correspondingly, each storage providers’s set of pledged sectors is partitioned into subsets, one subset for each window. Within a given window, each storage provider must submit a PoSt for each sector in their respective subset. For each day a storage provider is inactive it will receive a [fault fee](#penalty).
+_PoSt (Proof-of-Spacetime)_ window checks are performed on 24 hour intervals across the network to ensure that storage providers are continuing to host their required sectors as normal. Correspondingly, each storage provider’s set of pledged sectors is partitioned into subsets, one subset for each window. Within a given window, each storage provider must submit a PoSt for each sector in their respective subset. For each day a storage provider is inactive it will receive a [fault fee](#penalty).
```

Comment:
```suggestion
_PoSt (Proof-of-Spacetime)_ window checks are performed at 24 hour intervals across the network to ensure that storage providers are continuing to host their required sectors as normal. Correspondingly, each storage provider’s set of pledged sectors is partitioned into subsets, one subset for each window. Within a given window, each storage provider must submit a PoSt for each sector in their respective subset. For each day a storage provider is inactive it will receive a [fault fee](#penalty).
```